app/src/routes/v1/acronyms.js

```diff
@@ -51,6 +51,28 @@ acronymsRouter.get('/:acronym/clients', [
   }
 });
 
+/** Returns users from KC for the supplied acronym*/
+acronymsRouter.get('/:acronym/users', async (req, res) => {
+  // Check for required permissions. Can only fetch details for the acronyms you are associated with
+  const permissionErr = await permissionHelpers.checkAcronymPermission(req.kauth.grant.access_token.content.sub, req.params.acronym);
```

Comment:
are we doing this kind of logic elsewhere?
we should move into middleware.

Reply:
Yeah it's done in a number of places, that's why I refactored it out to the permissionhelpers util file a while back quickly.
Always intended on having it middleware fxn if possible, you're right that's the place for it I think. We can look at that for sure with next API changes.